<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFullscreen"/>
    <addaction name="actionAnimation"/>
    <addaction name="actionOriginal_Position"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionInformation"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_map"/>
    <addaction name="actionSave_map_2"/>
    <addaction name="actionSave_map"/>
    <addaction name="actionImport_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFeatures_Extract"/>
    <addaction name="actionIndexing"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionCapture_screen"/>
    <addaction name="actionPlugins"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionNormal"/>
    <addaction name="actionSOM_4"/>
    <addaction name="actionRearrangement"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About glMiSOM...</string>
   </property>
   <property name="toolTip">
    <string>Acerca de Visualizador de Mol√©culas</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionColor_Histogram">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Color Histogram</string>
   </property>
  </action>
  <action name="actionHu_Moments">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hu Moments</string>
   </property>
  </action>
  <action name="actionMatrix_Cooccurrences">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Matrix Cooccurrences</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Animation</string>
   </property>
  </action>
  <action name="actionTraining">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Training...</string>
   </property>
  </action>
  <action name="actionNormal_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionSOM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SOM</string>
   </property>
  </action>
  <action name="actionSOM_rearranged">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SOM rearranged</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionOriginal_Position">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Original Position</string>
   </property>
  </action>
  <action name="actionFeatures_Extract">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Features Extract...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionIndexing">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Indexing...</string>
   </property>
  </action>
  <action name="actionLoad_map">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load map...</string>
   </property>
  </action>
  <action name="actionSave_map">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save map as...</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search...</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionSOM_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SOM</string>
   </property>
  </action>
  <action name="actionRearrangement">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rearrangement</string>
   </property>
  </action>
  <action name="actionSOM_3">
   <property name="text">
    <string>SOM</string>
   </property>
  </action>
  <action name="actionSOM_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SOM</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionCapture_screen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Capture screen...</string>
   </property>
  </action>
  <action name="actionPlugins">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
  <action name="actionImport_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionSave_map_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save map...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
